torch/csrc/jit/passes/frozen_ops_to_mkldnn.cpp

```diff
@@ -456,6 +510,11 @@ void ComputeSubgraphInMKLDNN(Node* subgraph_node) {
       body_node->replaceInput(1, node->outputs().at(1));
     }
 
+    if (body_node->kind() == Symbol::aten("hardswish")) {
+      body_node->replaceWithNewSymbol(Symbol::aten("MKLDNNHardSwish"));
+      body_node->destroy();
```

Comment:
We should be adding `hardswish_` here https://github.com/pytorch/pytorch/blob/ce32d3d54b4bad74ecc4b44f68b1ca94b734e7ef/torch/csrc/jit/passes/frozen_ops_to_mkldnn.cpp#L819